src/Air.zig

```diff
@@ -973,13 +976,17 @@ pub const Inst = struct {
         _,
     };
 
+    comptime {
+        // Ensure no tags are added to InternPool without being added here
+        assert(@typeInfo(Ref).Enum.fields.len == @typeInfo(InternPool.Index).Enum.fields.len);
+    }
```

Comment:
Which logic is this protecting? This is redundant with

https://github.com/ziglang/zig/blob/6bd54793060191ffce2c7492a90a40a30f9e2a1d/src/Module.zig#L39-L49

Reply:
Whoops, I did look around for existing checks of this but didn't expect it to be in Module. Will remove my check